athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/metadata/optimizations/DataSourceOptimizations.java

```diff
@@ -72,6 +73,16 @@ public Map.Entry<String, List<OptimizationSubType>> withSupportedSubTypes(Pushdo
             }
             return new SimpleImmutableEntry<String, List<OptimizationSubType>>(SUPPORTS_COMPLEX_EXPRESSION_PUSHDOWN.getOptimization(), Arrays.stream(subTypesList).map(pushdownSubTypes -> new OptimizationSubType(pushdownSubTypes.getSubType(), pushdownSubTypes.getProperties())).collect(Collectors.toList()));
         }
+    },
+    DATA_SOURCE_HINTS("data_source_hints")
```

Comment:
Can you please add a comment of purpose on this parent type

Reply:
done;